docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   node_1:
-    image: ${SCYLLA_IMAGE}
+    image: scylladb/scylla-nightly
```

Comment:
This is bad idea, this CICD needs to be stable.

Reply:
I think that `scylla-enterprise-nightly:latest-enterprise` is more stable and still allows for testing the newest release without changing docker tags in every PR.

Reply:
This is perfect example of why you should not run cicd on nightly builds:
https://github.com/scylladb/gocql/actions/runs/9346882983/job/25722658483?pr=186

It is failing because image is broken, now you either ignore pipeline failure, or wait till working image is published.

Reply:
I will create separate PR to use newer version of SCYLLA_IMAGE for all of the nodes and I will bump scylla version in github workflows